blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Simplifying Firmware Updates in Dasharo'
+abstract: "Updating your firmware is an important part of keeping your device
+          secure and reliable. Making this process as easy and reliable as
+          possible is, therefore, a big focus for Dasharo. In this article we'll
+          dive into the latest feature that make that process a bit easier, and
```

Comment:
```suggestion
          dive into the latest feature that makes the update process a bit easier, and
```

Reply:
Fixed in eb660858d2cc0c15dfd4a7cce6caee520ad9831a